dessia_common/workflow/core.py

```diff
@@ -389,18 +390,18 @@ def __init__(self, blocks, pipes, output, *, imposed_variable_values=None,
 
         self.block_selectors = {}
         for i, block in enumerate(self.blocks):
```

Comment:
idx instead of i

Reply:
I cannot find the docs on it again, but as long as the convention didn't change, i is allowed by it and its use is even encouraged. Just like we should use x as a lambda variable or one-letter var in comprehension list. 

If you find some resources stating otherwise (because I cannot seem to find anything), I would gladly have a look at it, but I don't think we will initiate these kind of changes because we use the "i" absolutely everywhere

Reply:
Actually, pylint was set not to be happy when the iterator is only one letter long.
I guess this option has since been disabled :)